scripts/validate_docstrings.py

```diff
@@ -627,8 +636,8 @@ def validate_one(func_name):
             errs.append(error('SS03'))
         if doc.summary != doc.summary.lstrip():
             errs.append(error('SS04'))
-        elif (doc.is_function_or_method
-                and doc.summary.split(' ')[0][-1] == 's'):
+        elif (doc.is_function_or_method and
+                doc.summary.split(' ')[0][-1] == 's'):
```

Comment:
As said, try to avoid unrelated changes, they make things more difficult. And in this case, if I remember correctly, the Python standard is to have the binary operator after the line break, as it was before.

Reply:
Sorry I was looking at the related code and my editor highlighted PEP8 warning (W503): line break before binary operator. I'll revert this to follow accepted convention.